src/resources/postcss/components/_view-selector.pcss

```diff
@@ -256,9 +256,12 @@
 		&:hover,
 		&:focus {
 
+			.tribe-events-c-view-selector__list-item-icon {
+				opacity: var(--opacity-hover);
```

Comment:
Do we need to update the var name here?

Reply:
see the latest commit, i think you're reviewing on an old diff.

Reply:
Yep, I was haha! :)